manifests/base/cdrs/event-source-crd.yaml

```diff
@@ -1,16 +1,15 @@
-# Define a "event source" custom resource definition
 apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
while we are at it, lets update the `apiVersion` to `apiextensions.k8s.io/v1` for all CRDs. We can later add openAPI validation in a separate PR.

Reply:
Done. FYI: I've had to bump the Kubernetes version requirement in `docs/installation` to v1.16, since that's when `v1` became available.